src/components/PurchaseOrder/OpenOrderConfirmationModal/OpenOrderConfirmationModal.css

```diff
@@ -0,0 +1,13 @@
+.openOrderConfirmationModalFooter {
+  display: flex;
+  justify-content: space-between;
+  flex-grow: 1;
+}
+
+.openOrderConfirmationModalFooterButton {
+  margin-bottom: 0;
+}
```

Comment:
do we really need the class? It seems for me in Buttom.css exist the same buttomMargin0 and component waits the props

Reply:
yep, it can be, thanks